client/src/game/states.ts

```diff
@@ -9,15 +9,17 @@ export const notificationSettings : NotificationOptions = {
     duration: 3000,
 }
 
+type LobbyState = 'inGame' | 'inLobby'
+
 // global states which can be accessed and modified from every component
 export const states = {
     // game socket is not a ref as it shouldn't update other components
     // when created or re-assigned
     gameSocket: undefined as (Socket | undefined),
-    lobby: undefined as (types.LobbyId | undefined),
+    lobbyId: undefined as (types.LobbyId | undefined),
+    lobbyState: ref<LobbyState | null>(null),
```

Comment:
Instead of having these both as nullable and then when wanting to use them having to check both of them, why not have a `lobby` state  of type `{ id: types.LobbyId, state: LobbyState } | null`?

Reply:
no